gluster/glusto-tests
# [Test] Add test to compile linux kernel with self-heal

Test Case:
1. Create a volume of any type, start it and mount it.
2. Perform linux untar on the mount point and wait for it to complete.
3. Bring down brick processes of volume.
4. Compile linux kernel on monut point.
5. Bring offline bricks back online.
6. Wait for heal to complete.
7. Check arequal checksum on bricks and make sure there is
    no data loss.
